src/test/java/teammates/test/cases/util/FieldValidatorTest.java

```diff
@@ -96,12 +96,12 @@ public void testGetValidityInfoForNonHtmlField_sanitizedInput_returnEmptyString(
 
     @Test
     public void testGetValidityInfoForNonHtmlField_unsanitizedInput_returnErrorString() {
-        String unsanitizedInput = "Invalid unsanitized input <>\\/'&";
+        String unsanitizedInput = "Invalid unsanitized input <>\"/'&";
         String testFieldName = "Inconsequential test field name";
         String actual = validator.getValidityInfoForNonHtmlField(testFieldName, unsanitizedInput);
         assertEquals("Invalid unsanitized input should return error string",
-                     "The provided Inconsequential test field name is not acceptable to TEAMMATES as it "
-                         + "cannot contain the following special html characters in brackets: (&lt; &gt; \\ "
+                     "The provided Inconsequential test field name is not acceptable to TEAMMATES as it"
+                         + "cannot contain the following special html characters in brackets: (&lt; &gt; &quot;"
```

Comment:
Can i modify the expected string using trim() to remove the trailing spaces?
Thank you :)